### PR TITLE
Feature/tour page image gallery fetch data

### PR DESCRIPTION
**Pull request by Ahmed-Fahmy-Dev**

### CODE REVIEW

**Comment by AlinaPalyvoda** [comment]
I have a comment only on the display of photos on the page.
Maybe it's worth displaying only one current photo, and not with part of the next one? And place the mini-gallery in the center?
image

**Comment by Ahmed-Fahmy-Dev** [comment]
> I have a comment only on the display of photos on the page. Maybe it's worth displaying only one current photo, and not with part of the next one? And place the mini-gallery in the center? image

@AlinaPalyvoda  I totally agree in case the format of the pics will stay as it is now.  but am not the owner of this component. it is Rustam and he left the project already. so i will leave this till the end.

**Comment by AlinaPalyvoda** [comment]
> > I have a comment only on the display of photos on the page. Maybe it's worth displaying only one current photo, and not with part of the next one? And place the mini-gallery in the center? image
> 
> @AlinaPalyvoda I totally agree in case the format of the pics will stay as it is now. but am not the owner of this component. it is Rustam and he left the project already. so i will leave this till the end.

ok, maybe I'll find time to fix it later

**Review by AlinaPalyvoda — APPROVED** [review]

File: client/src/features/Tour/components/ImageGallery/Swiper.scss [hunk 1]
```diff
@@ -56,7 +56,7 @@
 
 @media screen and (max-width: 1220px) {
   .c {
-    margin: 100px 20px 20px;
+    margin: 0 20px 20px 20px;
```
Comment (HIMDanny) [inline client/src/features/Tour/components/ImageGallery/Swiper.scss:59]:
```suggestion
    margin: 0 20px 20px; // top | left-rigth | bottom
```

File: client/src/store/slices/tourSlice/tourSlice.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-param-reassign */
```
Comment (HIMDanny) [inline client/src/store/slices/tourSlice/tourSlice.js:1]: no longer needed

**Review by HIMDanny — APPROVED** [review]

**Pull request merged**